<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1948</width>
    <height>992</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QTableWidget" name="WhiskyTabelle">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>376</width>
        <height>903</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Arial Black</family>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="midLineWidth">
       <number>0</number>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAsNeeded</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAsNeeded</enum>
      </property>
      <property name="sizeAdjustPolicy">
       <enum>QAbstractScrollArea::AdjustToContents</enum>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::SelectedClicked</set>
      </property>
      <attribute name="horizontalHeaderMinimumSectionSize">
       <number>180</number>
      </attribute>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>190</number>
      </attribute>
      <attribute name="verticalHeaderMinimumSectionSize">
       <number>22</number>
      </attribute>
      <attribute name="verticalHeaderDefaultSectionSize">
       <number>30</number>
      </attribute>
      <column>
       <property name="text">
        <string>Brennerei</string>
       </property>
       <property name="font">
        <font>
         <family>Arial Black</family>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Herkunft</string>
       </property>
       <property name="font">
        <font>
         <family>Arial Black</family>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
      </column>
     </widget>
    </item>
    <item>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <family>Arial Black</family>
          <pointsize>14</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Brennerei</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEdit">
        <property name="font">
         <font>
          <family>Arial Black</family>
          <pointsize>12</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_3">
        <property name="font">
         <font>
          <family>Arial Black</family>
          <pointsize>14</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Region</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineEdit_3">
        <property name="font">
         <font>
          <family>Arial Black</family>
          <pointsize>12</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_2">
        <property name="font">
         <font>
          <family>Arial Black</family>
          <pointsize>14</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Herkunft</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="lineEdit_2">
        <property name="font">
         <font>
          <family>Arial Black</family>
          <pointsize>12</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTableWidget" name="RumTabelle">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>361</width>
        <height>0</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Arial Black</family>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAsNeeded</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAsNeeded</enum>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::SelectedClicked</set>
      </property>
      <attribute name="horizontalHeaderMinimumSectionSize">
       <number>180</number>
      </attribute>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>180</number>
      </attribute>
      <column>
       <property name="text">
        <string>Name</string>
       </property>
       <property name="font">
        <font>
         <family>Arial Black</family>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Herkunft</string>
       </property>
       <property name="font">
        <font>
         <family>Arial Black</family>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
      </column>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1948</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Black</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <widget class="QMenu" name="menuDatei_einlesen">
    <property name="font">
     <font>
      <family>Arial Black</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="title">
     <string>Funktionen</string>
    </property>
    <addaction name="actionNeuen_Rum"/>
    <addaction name="actionNeuen_Whisky"/>
    <addaction name="actionSpirituose_bearbeiten"/>
    <addaction name="actionProgramm_beenden"/>
   </widget>
   <widget class="QMenu" name="menuSortierung">
    <property name="font">
     <font>
      <family>Arial Black</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="title">
     <string>Auswahl</string>
    </property>
    <addaction name="actionBestand"/>
    <addaction name="actionGetestet"/>
    <addaction name="actionAlle"/>
   </widget>
   <widget class="QMenu" name="menuSortierung_2">
    <property name="title">
     <string>Sortierung</string>
    </property>
    <addaction name="actionAlphabetisch"/>
    <addaction name="actionBewertung_aufsteingend"/>
    <addaction name="actionBewertung_absteigend"/>
   </widget>
   <addaction name="menuDatei_einlesen"/>
   <addaction name="menuSortierung"/>
   <addaction name="menuSortierung_2"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionProgramm_beenden">
   <property name="text">
    <string>Programm beenden</string>
   </property>
   <property name="font">
    <font>
     <family>Arial Black</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
  </action>
  <action name="actionNeuen_Rum">
   <property name="text">
    <string>Neuen Rum</string>
   </property>
   <property name="font">
    <font>
     <family>Arial Black</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
  </action>
  <action name="actionSpirituose_bearbeiten">
   <property name="text">
    <string>Spirituose bearbeiten</string>
   </property>
   <property name="font">
    <font>
     <family>Arial Black</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
  </action>
  <action name="actionBestand">
   <property name="text">
    <string>Bestand</string>
   </property>
  </action>
  <action name="actionGetestet">
   <property name="text">
    <string>Getestet</string>
   </property>
  </action>
  <action name="actionAlle">
   <property name="text">
    <string>Alle</string>
   </property>
  </action>
  <action name="actionAlphabetisch">
   <property name="text">
    <string>Alphabetisch</string>
   </property>
   <property name="font">
    <font>
     <family>Arial Black</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
  </action>
  <action name="actionBewertung_aufsteingend">
   <property name="text">
    <string>Bewertung aufsteingend</string>
   </property>
   <property name="font">
    <font>
     <family>Arial Black</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
  </action>
  <action name="actionBewertung_absteigend">
   <property name="text">
    <string>Bewertung absteigend</string>
   </property>
   <property name="font">
    <font>
     <family>Arial Black</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
  </action>
  <action name="actionNeuen_Whisky">
   <property name="text">
    <string>Neuen Whisky</string>
   </property>
   <property name="font">
    <font>
     <family>Arial Black</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
